benchmark_scripts/bench.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+DATADIR=$(pwd)
+
+module load mpi
+
+source quick.rc
+
+mkdir -p $DATADIR/benchmark/
+
+location=${1:-benchmarkinputs}
+
+for fileraw in $location/*
+do
+        file=$(basename $fileraw .in)
+        for number in 1 2 4 6 8 12 16
+        do
+                cp $location/$file.in $DATADIR/benchmark/$number$file.in
```

Comment:
Let us com up with a systematic naming of benchmark output files. I suggest something like `${benchmark}_cpu_${ntasks}.out`.

We do not need to rename the input file as long as we do not run two calculations with the same input file in the same location (otherwise two quick jobs would overwrite their outputs). In the batch script we could run in a local scratch directory. This also avoids timing inconsistencies in case there is heavy load on the shared home file system:
```
cd $SLURM_SCRATCH_DIR # this is defined on Aerosol and Expanse and automatically deleted after jobs finish
cp $SLURM_SUBMIT_DIR/benchmark.inp ./ # this assumes the input file is in the dir where we called the slurm script from, may need updating
mpirun -np NNN quick.MPI benchmark.inp
mv benchmark.out $SLURM_SUBMIT_DIR/$SLURM_JOB_NAME.out # assumes that we set the job name to benchmark_cpu_NNN.out
```